src/domUpdates.js

```diff
@@ -258,12 +258,14 @@ const createIngredientsHTML = ingredients => {
 const searchForRecipes = () => {
   const data = {
     ourRecipes: recipeData,
-    yourRecipes: null
+    yourRecipes: currentUser.recipesToCook
   }
   let searchedRecipes = searchRecipes(data[pageData.currentView], ingredientsData, searchBar.value)
-  searchedRecipes.length  
+  if(searchedRecipes) {
+    searchedRecipes.length  
```

Comment:
Given your approach, you want to try this?
if(searchedRecipes?.length? renderGrid(searchedRecipes))

Reply:
Putting `?` after searchedRecipes makes it go to the else statement and shows no results for `''` searches and `' '` searches, but Matt had requested those searches do nothing. This approach makes sure they `do nothing` instead of going to the else statement.

Reply:
Aha. Makes sense